src/lib/viewers/BaseViewer.js

```diff
@@ -553,7 +553,8 @@ class BaseViewer extends EventEmitter {
      * @protected
      * @return {void}
      */
-    toggleFullscreen() {
+    toggleFullscreen(isFullscreen, fullscreenToggleEl) {
+        this.fullscreenToggleEl = fullscreenToggleEl;
```

Comment:
Is this a new class property? Can it be documented at the top of the file with all the others?